gulp/typescript.js

```diff
@@ -7,33 +7,40 @@ module.exports = (blueprint, gulp, plugins) => {
     const mergeStream = require("merge-stream");
     const path = require("path");
 
-    function createTypescriptProject(tsConfigPath) {
-        return plugins.typescript.createProject(tsConfigPath, {
+    blueprint.defineTaskGroup({
+        block: "all",
+        name: "tslint",
+    }, (project, taskName) => {
+        gulp.task(taskName, () => (
+            gulp.src([
+                path.join(project.cwd, "!(dist|node_modules|typings)", "**", "*.{js,jsx,ts,tsx}"),
+                // exclude nested dist directories (ex: table/preview/dist)
+                "!" + path.join(project.cwd, "*", "dist", "**", "*.{js,jsx,ts,tsx}"),
```

Comment:
no, it's always been there. it's actually due to table's webpack-based build which doesn't rely on these changes even a little bit.